ee/clickhouse/models/cohort.py

```diff
@@ -1,15 +1,33 @@
-from typing import Any, Dict, Tuple
+import uuid
+from typing import Any, Dict, List, Optional, Tuple
 
 from ee.clickhouse.client import sync_execute
 from ee.clickhouse.models.action import format_action_filter
 from ee.clickhouse.sql.cohort import CALCULATE_COHORT_PEOPLE_SQL
-from ee.clickhouse.sql.person import GET_LATEST_PERSON_ID_SQL, GET_PERSON_IDS_BY_FILTER
+from ee.clickhouse.sql.person import (
+    GET_LATEST_PERSON_ID_SQL,
+    GET_PERSON_IDS_BY_FILTER,
+    INSERT_PERSON_STATIC_COHORT,
+    PERSON_STATIC_COHORT_TABLE,
+)
+from ee.idl.gen import person_static_cohort_pb2
+from ee.kafka_client.client import ClickhouseProducer
+from ee.kafka_client.topics import KAFKA_PERSON_STATIC_COHORT
 from posthog.models import Action, Cohort, Filter, Team
 
 
 def format_person_query(cohort: Cohort) -> Tuple[str, Dict[str, Any]]:
     filters = []
     params: Dict[str, Any] = {}
+
+    if cohort.is_static:
+        return (
+            "person_id IN (SELECT person_id FROM {} WHERE cohort_id = {} AND team_id = {})".format(
```

Comment:
last point about this is that we should be using params  `%(cohort_id)s` and `%(team_id)s` rather than formatting in the id. shouldn't really be an issue here but for consistency and because this function already is setup to return params